R/dataframe-frame.R

```diff
@@ -1316,3 +1316,47 @@ dataframe__partition_by <- function(..., maintain_order = TRUE, include_key = TR
       lapply(\(ptr) .savvy_wrap_PlRDataFrame(ptr) |> wrap())
   })
 }
+
+#' @inherit lazyframe__rolling title description params
+#'
+#' @return [RollingGroupBy][RollingGroupBy_class] (a DataFrame with special
+#' rolling groupby methods like `$agg()`).
```

Comment:
Currently it automatically opens the docs in `polars` so those should be added in neopolars too